enterprise/reporting/reporting.go

```diff
@@ -89,58 +90,59 @@ func NewFromEnvConfig(log logger.Logger) *HandleT {
 		whActionsOnly:             whActionsOnly,
 		sleepInterval:             sleepInterval,
 		mainLoopSleepInterval:     mainLoopSleepInterval,
+		region:                    config.GetString("region", ""),
 	}
 }
 
-func (handle *HandleT) setup(beConfigHandle backendconfig.BackendConfig) {
```

Comment:
if you are renaming `r`, won't it be better to rename it to `h` instead of `handle`? Since, such a large receiver name is avoided in golang.

Reply:
Used `r` to just represent reporting handle... can change it to `rh` though